src/mpi/init/mpi_init.h

```diff
@@ -37,6 +37,7 @@
         1: Print internal summary of various debug information, such as memory allocation by category.
         Each layer may print their own summary information. For example, ch4-ofi may print its provider
         capability settings.
+        2: Print the preferred NIC for each rank
```

Comment:
You mean "Also print the preferred NIC for each rank"?

Reply:
Done.